leobot_description/urdf/wheels.urdf.xacro

```diff
@@ -4,27 +4,37 @@
     <xacro:macro name="standard_wheel" params="prefix direction bearing_radius radius width mass base_radius">
         <link name="${prefix}_bearing_link">>
             <visual>
+
                 <xacro:sphere_geometry radius="${bearing_radius}"/>
                 <xacro:blue_material/>
+
             </visual>
             <xacro:sphere_collision radius="${bearing_radius}"/>
             <xacro:sphere_inertia mass="0.1" radius="${bearing_radius}"/>
         </link>
         <joint name="${prefix}_bearing_base_joint" type="fixed">
-            <origin rpy="0 0 0" xyz="0.0 ${direction * (base_radius + bearing_radius - 0.0001)} ${-(base_height/2 - bearing_radius)}"/>
+            <origin rpy="0 0 0" xyz="0.35 ${direction * (base_radius + bearing_radius - 0.0001)} 0.04"/>
```

Comment:
What is 0.35 and 0.04? Try to avoid hardcoded values and calculate them from parameters or properties.